src/test/unit/controls/Bubble/BubblePanning-spec.js

```diff
@@ -126,6 +126,28 @@ describe("Bubble - Panning", () => {
             );
             expect(bubbleContent.length).toEqual(3);
         });
+        it("Dynamic data is updated when there is no data", () => {
```

Comment:
Here, initially, graph loads with some data and the legend get displayed properly. And on click of panning button, the legend does not hold any data, as a result, the dynamic data is updated with no data view and legend gets disabled.

Reply:
nit: It can be updated all related test cases
```js
describe("when there is no data", () => {
     it("should update the dynamic data and disable the legend", () => {

    });
});
``

Reply:
Moved [here](https://github.com/cerner/carbon-graphs/pull/289/commits/35238c46e904c1ea45e59bcc1f7cc4c915fcd510)